plugins/evernote.koplugin/main.lua

```diff
@@ -111,6 +125,129 @@ function EvernoteExporter:addToMainMenu(menu_items)
+                                        {
+                                            text = _("Cancel"),
+                                            callback = function()
+                                                UIManager:close(url_dialog)
+                                            end
+                                        },
+                                        {
+                                            text = _("OK"),
+                                            callback = function()
+                                                local fields = url_dialog:getFields()
+                                                local ip = fields[1]
+                                                local port = tonumber(fields[2])
+                                                if ip ~= "" then
+                                                    if port and port < 65355 then
+                                                        self.joplin_IP = ip
+                                                        self.joplin_port = port
+                                                    end
+                                                    self:saveSettings()
+                                                end
+                                                UIManager:close(url_dialog)
+                                            end
+                                        }
+                                    }
+                                }
+                            }
+                            UIManager:show(url_dialog)
+                            url_dialog:onShowKeyboard()
+                        end
+                    },
+                    {
+                        text = _("Set authorization token"),
+                        keep_menu_open = true,
+                        callback = function()
+                            local MultiInputDialog = require("ui/widget/multiinputdialog")
+                            local auth_dialog
+                            auth_dialog = MultiInputDialog:new{
+                                title = _("Set authorization token for Joplin"),
+                                fields = {
+                                    {
+                                        text = self.joplin_token,
+                                        input_type = "string"
+                                    }
+                                },
+                                buttons = {
+                                    {
+                                        {
+                                            text = _("Cancel"),
+                                            callback = function()
+                                                UIManager:close(auth_dialog)
+                                            end
+                                        },
+                                        {
+                                            text = _("OK"),
```

Comment:
```suggestion
                                            text = _("Set token"),
```

Sorry, I overlooked this before.